client/service/serviceAPI.js

```diff
@@ -0,0 +1,28 @@
+const serviceAPI = {
```

Comment:
@IgorChaikin Could we get rid of this `service` description, for example you can send query directly from redux action, or you can create small `api.js` file, where you can describe `.get()`, and `.post` methods, and use something like:

```js
export const postOrder = createAsyncThunk('CREATE_ORDER', async (data) => {
  const { data } = await api.post('/api/orders', data);
  return data;
});
```